tf_big/cc/kernels/big_kernels.cc

```diff
@@ -81,11 +82,49 @@ class BigAddOp : public OpKernel {
     Tensor* output;
     OP_REQUIRES_OK(ctx, ctx->allocate_output(0, TensorShape{}, &output));
 
-    MatrixXm res = val1->value + val2->value;
-    BigTensor big(res);
+    auto res = *val1 + *val2;
```

Comment:
how is data moved around here?

Reply:
Can you rephrase your question? Not sure what you mean.